CHANGELOG.md

```diff
@@ -20,6 +20,295 @@ For reference, the possible headings are:
 
 ## [Unreleased]
 
+## [0.7.0] - 2020-03-25
+
+- PB-584 Update the manifest Cargo.toml with description, license, keywords, repository URL and project homepage (#343) [Ricardo Saffi Marques]
```

Comment:
Wow. This is so big that I wonder if it's really informative. Should be sum it up by saying the coordinator was re-implemented from scratch, and that this release is _obviously_ a breaking change?

Reply:
For the Github release I did put a disclaimer at the beginning: 

https://github.com/xainag/xain-fl/releases/tag/v0.7.0

Reply:
I like it :+1: Maybe the disclaimer would be enough for this release?

Reply:
I'm fine with updating the PR and adding only the disclaimer, yes.

Reply:
Updated